assets/scss/local.scss

```diff
@@ -1,11 +1,3 @@
 .govuk-main-wrapper {
   min-height: 300px;
 }
-
-.bookings-content-block > p, .bookings-content-block > h3 {
```

Comment:
Rather than extend this to style `h2`s too, removing in favour of a few GOVUK margin override classes - simpler markup overall.